ddtrace/tracer/tracer.go

```diff
@@ -91,6 +91,9 @@ func Start(opts ...StartOption) {
 	if internal.Testing {
 		return // mock tracer active
 	}
+	if !boolEnv("DD_TRACE_ENABLED", true) {
```

Comment:
I can already see comments like 
> I enabled `DD_TRACE_ENABLED=true` but I don't see any traces

🙂

Reply:
We can decline to implement `DD_TRACE_ENABLED`. I think it's not a great fit for the Go tracer.